examples/data-objects/monaco/src/chaincode.ts

```diff
@@ -92,17 +76,12 @@ export class MonacoRunner extends DataObject implements
      */
     private codeEditor: monaco.editor.IStandaloneCodeEditor;
 
-    public render(elm: HTMLElement, options?: IFluidHTMLOptions): void {
-        if (!this.mapHost) {
-            this.mapHost = document.createElement("div");
-            elm.appendChild(this.mapHost);
-            this.initializeEditorDiv().catch((error) => { console.error(error); });
-        } else {
-            if (this.mapHost.parentElement !== elm) {
-                this.mapHost.remove();
-                elm.appendChild(this.mapHost);
-            }
-        }
+    public mount(elm: HTMLElement): void {
+        MonacoRunnerView(elm, this);
+    }
+
+    public unmount() {
+        console.log("unmount");
```

Comment:
If implementing `IFluidMountableView` you must provide a real `unmount` implementation.

To avoid sprinkling too much view management into `MonacoRunner`, you might consider instead choosing to make `MonacoRunner` implement `IProvideFluidMountableView` rather than `IFluidMountableView`, such that you can partition the view management and mount/unmount logic to a separate view class to be returned by `IFluidMountableView`.